tools/clang/test/DXC/ftime-trace-granularity.hlsl

```diff
@@ -0,0 +1,7 @@
+// RUN: %dxc -E main -T vs_6_0 %s -ftime-trace | FileCheck %s
+// RUN: %dxc -E main -T vs_6_0 %s -ftime-trace=%t.json -ftime-trace-granularity=0
```

Comment:
I don't think there's anything actually testing that the option does anything?

Suggestion: try setting it to some super larger value which means that all the trace events get filtered out.

Reply:
Will do!